example/kaggle-ndsb1/gen_img_list.py

```diff
@@ -18,10 +18,10 @@
 from __future__ import print_function
 import csv
 import os
-import sys
 import random
 import numpy as np
 import argparse
+from six.moves import xrange
```

Comment:
just replace xrange with `range` for python3 compatibility, since there's no performance issue here.